src/explore-education-statistics-admin/src/components/ReleaseServiceStatus.tsx

```diff
@@ -12,19 +12,24 @@ interface Props {
   refreshPeriod?: number;
   exclude?: 'status' | 'details';
   isApproved?: boolean;
+  newAdminStyle?: boolean; // EES-3217 CLEANUP
```

Comment:
As suggested in another comment for scheduled releases, we should move this component's logic into a hook and may not need this flag.